doc/src/quickstart.md

```diff
@@ -251,42 +251,42 @@ errors and have the system recover automatically:
 ```
 >>> 2 + + 3
 Parsing error at line 1 column 5. Repair sequences found:
-   1: Delete +
-   2: Insert INT
-Result: 5
+   1: Insert INT
```

Comment:
For this one, either order of repair sequences is correct, but I feel that explaining to users the `Err` case is more confusing than the `Delete +` case, so I'd be inclined to keep the original ordering in this one case.

Reply:
Fixed by reverting back to the original repair sequence in [892cc71](https://github.com/softdevteam/grmtools/pull/366/commits/892cc71bbbae9eb24f9da69a547b80167198ef1d)